tests/integration/components/ember-attacher/hide-on-blur-test.js

```diff
@@ -72,7 +72,13 @@ module('Integration | Component | hideOn "blur"', function(hooks) {
 
     assert.equal(isVisible(attachment), true, 'Now shown');
 
-    await focus('#attachment-focus-me');
+    const targetInsideAttachment = find('#attachment-focus-me')
+    await focus(targetInsideAttachment)
+    await waitUntil(() => {
+      if (document.activeElement === targetInsideAttachment) return true
+
+      focus(targetInsideAttachment)
+    });
```

Comment:
Sometimes the "focus" event didn't work for the firs time that caused the flaky tests here